cypress/fixtures/addcase.json

```diff
@@ -1,3 +1,14 @@
 {
-  "address": "1018vn113"
+  "queryString": "1018vn113",
+  "street": "Weesperstraat 113",
+  "zipCode": "1018VN",
+  "city": "Amsterdam",
+  "sia_identification": "123456",
+  "sia_description": "Test samanvatting melding",
+  "description": "Test toelichting",
+  "reporter_name": "Naam reporter",
```

Comment:
camelCase or snake_case? There is no standard for JSON, but we have to choose one for our project. Let's use camelCase.